src/Concerns/HasIntervals.php

```diff
@@ -49,7 +49,7 @@ trait HasIntervals
      * @return $this
      * @throws \Exception
      */
-    public function interval($start, $stop = null)
+    public function interval($start, $stop = null): self
```

Comment:
Waarom niet `addInterval`?

Reply:
Dat zou sowieso een breaking change zijn in *elke* query welke er bestaat. Verder wordt er nergens een "add" prefix gebruikt in de methodes welke gebruikt worden in de "public methods". Ik zie ook niet in waarom dat logischer zou zijn.